src/cmdline.rs

```diff
@@ -334,6 +341,13 @@ pub fn parse_args() -> Result<Config> {
                         .long("preserve-on-error")
                         .help("Don't clear partition table on error"),
                 )
+                .arg(
+                    Arg::with_name("http-retries")
+                        .long("http-retries")
+                        .value_name("N")
+                        .help("HTTP retries, or string \"infinite\"")
```

Comment:
Before someone suggests making `-1` mean infinite: not opposed, but I personally think this is much more explicit and self-documenting. There's also precedence with `sleep infinity`.